src/services/location.js

```diff
@@ -221,37 +305,37 @@ ngeo.Location.prototype.getParamKeysWithPrefix = function(prefix) {
  * @export
  */
 ngeo.Location.prototype.getFragmentParamKeysWithPrefix = function(prefix) {
-  return this.getFragmentParamKeys().filter(key => key.indexOf(prefix) == 0);
+  const keys = [];
```

Comment:
What about an helper function to create an array out of on object, optionaly passing a filter?